scanner/src/main/kotlin/scanners/ScanCodeResultParser.kt

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright (C) 2019 Bosch Software Innovations GmbH
```

Comment:
This should be "Copyright (C) 2020 Bosch.IO GmbH", but before that please also add "Copyright (C) 2017-2019 HERE Europe B.V." as this is mostly moved code.